cleannrooster/druids
# Updates and fixes

* update to latest Spell Engine/Power
* call correct tinyconfig, should fix #1 
* optimize all imports
* remove some unused spellblades code
* prevent crystal staff recipe from loading if betterend isn't loaded
* replace fabric template gen icon.png with the one from curseforge (resized)
* update lang file damage placeholders
